doc/manual/customizing.rst

```diff
@@ -276,6 +299,20 @@ Other keywords that can be used in a spec:
 * To prevent a control icon from appearing in the Operations plugin
   manager tray specify `optray=False`.
 
+=================
+Disabling Plugins
+=================
+
+Both local and global plugins can be disabled (thus, not shown in the
+reference viewer) using the ``--disable-plugins`` option in the
+command line interface. To remove multiple plugins at once,
+use a comma-separated list. For example::
+
+    ginga --disable-plugins=Zoom,Compose
+
+Some essential plugins, like ``Info``, cannot be disabled (i.e., attempt to
+disable them would have no effect).
```

Comment:
Actually, there is no problem to disable `Info`, `Pan`, `Cursor`, `Colorbar`, `Toolbar` or basically any plugin.  You can even disable `Operations` and things still work (although maybe not as conveniently).

Reply:
@ejeschke are you sure? I put `disable_plugins = "Info"` in my `general.cfg` but I still see "Info" loaded.